src/device-registry/bin/jobs/test/ut_check-active-statuses.js

```
@@ -0,0 +1,97 @@
+const expect = chai.expect;
+const sinonChai = require("sinon-chai");
+
+const checkActiveStatuses = require("@bin/jobs/check-active-statuses");
+
+const DeviceModelMock = sinon.mock(DeviceModel);
+const logTextSpy = sinon.spy(console.log);
+const logObjectSpy = sinon.spy(console.log);
+const loggerErrorSpy = sinon.spy(logger.error);
+
+beforeEach(() => {
+  DeviceModelMock = sinon.mock(DeviceModel);
```

Biome

[error] Can't assign DeviceModelMock because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)